src/app.ts

```diff
@@ -22,5 +23,7 @@ export function buildApp() {
   addUploadFileEndpoint(app);
   addCheckFileExistsEndpoint(app);
   addCheckDirExistsEndpoint(app);
+  createADirEndpoint(app);
```

Comment:
Words like "A" and "The" should be avoided when naming things. `createDirEndpoint` is clear enough. The same should apply to:
- `createADir.test.ts` -> `createDir.test.ts`
- `createADir.ts` -> `createDir.ts`